package com.covielloDevs.SistemaDeVerificacion.utils.exceptions.usuario;

public class UsuarioNotFoundException extends RuntimeException {
    public UsuarioNotFoundException(String message) {
        super(message);
    }
}
